src/core/tsi/private_key_offload_util.cc

```diff
@@ -59,57 +54,46 @@ ToSignatureAlgorithmClass(uint16_t algorithm) {
     case SSL_SIGN_RSA_PSS_RSAE_SHA512:
       return CustomPrivateKeySigner::SignatureAlgorithm::kRsaPssRsaeSha512;
   }
+#endif  // OPENSSL_IS_BORINGSSL
   return absl::InvalidArgumentError("Unknown signature algorithm.");
 }
 
-void SetPrivateKeyOffloadingContextIndex(int index) {
-  g_ssl_ex_private_key_offloading_context_index = index;
-  GRPC_CHECK_NE(g_ssl_ex_private_key_offloading_context_index, -1);
-}
-
-int GetPrivateKeyOffloadingContextIndex() {
-  return g_ssl_ex_private_key_offloading_context_index;
-}
-
-void SetPrivateKeyOffloadFunctionIndex(int index) {
-  GRPC_CHECK_NE(index, -1);
-  g_ssl_ctx_ex_private_key_function_index = index;
-}
-
-int GetPrivateKeyOffloadFunctionIndex() {
-  return g_ssl_ctx_ex_private_key_function_index;
-}
-
+#if defined(OPENSSL_IS_BORINGSSL)
 void TlsOffloadSignDoneCallback(TlsPrivateKeyOffloadContext* ctx,
                                 absl::StatusOr<std::string> signed_data) {
+  LOG(ERROR) << "TlsOffloadSignDoneCallback hs_result_addr "
```

Comment:
Please remove logging statements, or move them into a tracer.  And they should definitely not be at log level ERROR.